src/pretix/presale/views/order.py

```diff
@@ -371,6 +371,7 @@ def get_context_data(self, **kwargs):
             order=self.order
         )
         ctx['tickets_with_download'] = [p for p in ctx['cart']['positions'] if p.generate_ticket]
+        ctx['user_change_allowed'] = self.position.user_change_allowed
```

Comment:
See comment above. As `OrderPositionDetails` is used on a per-ticket context of a user being an attendee, I’d change this to 

```suggestion
        ctx['user_change_allowed'] = self.position.attendee_change_allowed
```

One could keep `ctx['user_change_allowed']` as that way, the template `pretixpresale/event/position.html` could be reused.